Description: remove LICENSE from automake
 upstream contains LICENSE in Makefile.am which fails when
 automake is run during Debian build
Author: fossfreedom <foss.freedom@gmail.com>
Origin: other, N/A
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: rhythmbox-plugin-alternative-toolbar-0.15.0/Makefile.am
===================================================================
--- rhythmbox-plugin-alternative-toolbar-0.15.0.orig/Makefile.am	2015-12-21 13:11:50.312010742 +0000
+++ rhythmbox-plugin-alternative-toolbar-0.15.0/Makefile.am	2015-12-21 13:36:14.849855359 +0000
@@ -59,13 +59,11 @@
 	$(IMAGE_FILES) \
 	$(UI_FILES) \
 	$(top_srcdir)/po/Makefile.in.in \
-	schema/org.gnome.rhythmbox.plugins.alternative_toolbar.gschema.xml \
-	LICENSE
+	schema/org.gnome.rhythmbox.plugins.alternative_toolbar.gschema.xml 
 
 rb_plugin_lib_DATA = \
 	$(PLUGIN_FILES) \
-	alternative-toolbar.plugin \
-	LICENSE
+	alternative-toolbar.plugin 
 
 
 rb_plugin_imgdir = $(datadir)/rhythmbox/plugins/alternative-toolbar/img
